### ISSUE TITLE
question: valid jsx-indent warning or not

### ISSUE THREAD

**Issue by webOS101** [yannickcr/eslint-plugin-react#1092]
While testing some different scenarios with `jsx-indent` in master I discovered the following code throws a warning.  While ugly, it doesn't seem to violate the `jsx-indent` rule explicitly.  Would this be consider a valid warning:

```
function Hello (props) {
	return (
		<div
			className="hello"
		>Hello
		</div>
	);
}
```
`warning  Expected indentation of 3 tab characters but found 0`
(on the line with `>Hello`)

**Comment by ljharb** [comment]
Indeed, it should be:
```jsx
function Hello (props) {
	return (
		<div
			className="hello"
		>
			Hello
		</div>
	);
}
```

or

```jsx
function Hello (props) {
	return (
		<div className="hello">
			Hello
		</div>
	);
}
```

**Comment by webOS101** [comment]
fair enough, I would prefer either of those as well.  closing this.
